frontend/talentsearch/src/js/components/Browse/OngoingRecruitmentSection.tsx

```diff
@@ -613,6 +625,13 @@ export const OngoingRecruitmentSection = ({
     ? streamsWithAvailablePools.filter((s) => s.key === quickFilterStream)
     : streamsWithAvailablePools;
 
+  // sort list so that recommended streams come first
+  streamsToShow.sort((s1, s2) =>
+    !streamIsRecommended(s1, mySkillIds) && streamIsRecommended(s2, mySkillIds)
+      ? 1
+      : 0,
+  );
```

Comment:
Sorting does not seem to be working for me.

<img width="1159" alt="Screenshot 2022-12-13 at 12 59 48 PM" src="https://user-images.githubusercontent.com/4127998/207409721-35804b94-23d9-43d3-bed6-87266fe6c37a.png">

Reply:
Wow, it turns out the sorting works quite differently between Firefox and Chrome.  If you don't do it properly you can get really different results!  :laughing: 
Fixed in [fix sorting](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5102/commits/0f16f15bc16dbfb2b8018a6e3c6bf6ed6a9a8cbb)